python_modules/libraries/dagster-snowflake/dagster_snowflake/resources.py

```diff
@@ -94,7 +112,36 @@ def get_connection(self, raw_conn=True):
                 conn.commit()
             conn.close()
 
-    def execute_query(self, sql, parameters=None, fetch_results=False, use_pandas_result=False):
+    @public
+    def execute_query(
+        self,
+        sql: str,
+        parameters: Optional[Mapping[Any, Any]] = None,
```

Comment:
TODO - see if i can make this type annotation more specific - need to dig into snowflake docs